### ISSUE TITLE
Add parataxis.profile

### ISSUE THREAD

**Issue by n01r** [ComputationalRadiationPhysics/parataxis#14]
A `parataxis.profile.example` should be added to `parataxis/submit/hypnos-hzdr/` which can be used as a reference for the user to create their own `parataxis.profile` which should be sourced by the `.tpl` files during a cluster job.

So far the `.tpl` files try to source a `picongpu.profile` which is not mentioned as a prerequisite in the documentation. That can also lead to clashes if you are also working with PIConGPU since the dependencies of the latter and ParaTAXIS will most likely differ.

**Comment by Flamefire** [comment]
Especially in reference to #12: There is a default fallback to the picongpu.profile in the TBGs (https://github.com/ComputationalRadiationPhysics/parataxis/blob/ee10cd91cb54ad314bacdff4d01f5f5909ec6b6d/submit/bash/bash_mpiexec.tpl#L38)
You can use your own during the tests via the `-p` switch (https://github.com/ComputationalRadiationPhysics/parataxis/blob/master/buildSystem/runTests.py#L179)
It does need documentation though.

**Comment by ax3l** [comment]
Be aware we modernized how PIConGPU selects `.profile` files on `tbg` submit time be be more flexible, you might want to add this to parataxis: https://github.com/ComputationalRadiationPhysics/picongpu/pull/1975

**Comment by n01r** [comment]
Right, that's probably why I was only able to run interactively: it loaded the wrong profile

**Comment by Flamefire** [comment]
I choose to use a TBG var instead of an env var so it appears in the result. This makes it possible to see, which profile you used later. But feel free to change it to the picongpu version which needs a change in the test script too to set an env var instead
